wordpress/README.md

```diff
@@ -4,9 +4,18 @@ Connects Cohere to Wordpress.
 
 ## Configuration
 
-This connector requires that the environment variables `WORDPRESS_USERNAME`, `WORDPRESS_PASSWORD`, and `WORDPRESS_CONNECTOR_API_KEY` be set in order to run. These variables can optionally be put into a `.env` file for development.
+This connector requires that the environment variables `WORDPRESS_USERNAME`, `WORDPRESS_PASSWORD` be set in order to run. These variables can optionally be put into a `.env` file for development.
```

Comment:
What about `WORDPRESS_URL`? Would be helpful to add a precision as to what this value should contain

Reply:
Fixed, thank you